### PR TITLE
Laboratory work #1, Evgeni Baronov - 22FPL2

### PR DESCRIPTION
**Pull request by Baronofff**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('woow')
```
Comment (marina-kaz) [inline main.py:1]: Hello! Welcome to the course!
Please do not push any extra files and work in the folder of the first lab.
Thank you!

**Review by marina-kaz — CHANGES_REQUESTED** [review]
Not much to review today. Time to speed up :)

**Review by marina-kaz — CHANGES_REQUESTED** [review]
No changes to review

**Comment by EIzharikov** [comment]
Hello! Pay attention to the letter "в". There is probably an error in decoding, because there must be another one. 
You will succeed, good luck!

File: lab_1_classify_by_unigrams/main.py [hunk 2]
```diff
@@ -120,27 +110,16 @@ def detect_language(
...
-        return str(profile_2['name'])
-    if profile_1_metric < profile_2_metric:
-        return str(profile_1['name'])
-
-    return [profile_1['name'], profile_2['name']].sort()
+    if compare_profiles(unknown_profile, profile_1) < compare_profiles(unknown_profile, profile_2):
+        return profile_1["name"]
+    elif compare_profiles(unknown_profile, profile_1) > compare_profiles(unknown_profile, profile_2):
+        return profile_2["name"]
+    else:
```
Comment (marina-kaz) [inline lab_1_classify_by_unigrams/main.py:120]: Using `else` after the return statement is redundant.

**Review by marina-kaz — COMMENTED** [review]
Some fixes and you are ready for your hand-in.

**Pull request closed without merging**